scripts/run-postgresql-dev.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export PG_CONTAINER_NAME="postgres"
+PG_IMAGE_NAME="postgres:9.5"
+PG_HOST_DATA="/tmp/pgdata"
+PG_CONTAINER_DATA="/var/lib/postgresql/data"
+PG_USER="iroha"
+PG_PASS="helloworld"
```

Comment:
Change PG_PASS to "mysecretpassword".
Since the script is designed for development needs, the password should be changed.

https://github.com/hyperledger/iroha/blob/38dd0a1818e0269834625985955a360d34cb8baf/example/config.sample#L5

https://github.com/hyperledger/iroha/blob/38dd0a1818e0269834625985955a360d34cb8baf/example/config.docker#L5

Tests also use mysecretpassword by default:

https://github.com/hyperledger/iroha/blob/38dd0a1818e0269834625985955a360d34cb8baf/test/framework/config_helper.hpp#L27